cluster/gce/gci/configure-helper.sh

```diff
@@ -1561,7 +1564,7 @@ if [[ "${KUBERNETES_MASTER:-}" == "true" ]]; then
   create-master-etcd-auth
 else
   create-node-pki
-  create-kubelet-kubeconfig
+  create-kubelet-kubeconfig ${KUBERNETES_MASTER_NAME}
```

Comment:
Before we used two different values for the apiserver address, `KUBERNETES_MASTER_NAME` and `KUBELET_APISERVER`. Have you tested that this works when the master kubelet is and isn't registered into the cluster?